### PR TITLE
[Legal] - Welcome, ToS and PP updates.

### PR DESCRIPTION
**Pull request by TheoTechnicguy**
! I would like the PR #21 to be push ASAP.
**The legal basis will not be valid until** ***at least the welcome message is pushed***!

### CODE REVIEW

**Comment by BhasherBEL** [comment]
The license does not seem to be accessible

**Comment by TheoTechnicguy** [comment]
Fix: the licence should now be accessible.

File: src/publics/css/legal.less [hunk 1]
```diff
@@ -1,36 +1,70 @@
 section{
```
Comment (TheoTechnicguy) [inline src/publics/css/legal.less:1]: Just formating - no functional changes.

**Review by TheoTechnicguy — COMMENTED** [review]
For me, these changes are reviewed.

**Pull request merged**